client/src/Components/CommonComponent/Table/Table.jsx

```diff
@@ -230,32 +263,32 @@ class TableCmponent extends Component {
               key="options"
               render={(text, record) => (
                 <span>
-                    <Icon onClick = {this.props.viewValues("singleCustomer", "viewVisibility", record.key, record)}
-                      style={{
-                        fontSize: "1.2rem",
-                        color: "rgba(0, 0, 0, 0.65)"
-                      }}
-                      type="profile"
-                    />
+                  <Icon onClick={this.props.viewValues("singleCustomer", "viewVisibility", record.key, record)}
```

Comment:
@naremanhilles I think you have to modify the onClick because right now, you are actually calling the viewValues function during this render and using its return value as the onClick, so I think this won't work right now.
Instead, you could use an arrow function:
`<Icon onClick={() => this.props.viewValues("singleCustomer", "viewVisibility", record.key, record)}`
Though arrow functions inside render are a little bit inefficient so it would be even better if you put the click handler into a class function and then used that:
```
//somewhere else in the file:
handleClickViewIcon = () => this.props.viewValues("singleCustomer", "viewVisibility", record.key, record)

//and then inside render:
<Icon onClick={this.handleClickViewIcon}
```